### PR TITLE
Added arm_id variable to use moveit for multiple panda arms

### PR DESCRIPTION
**Pull request by Xiaoyu27**
Hi Moveit group,

Please check the modification I made on the panda_moveit_config. I added the "arm_id" to the files. Now each panda arm can has its own move_group. It is possible to add multiple panda arms easily now.

Best,
Xiao

### CODE REVIEW

**Comment by rhaschke** [comment]
Generally this looks good. Please rebase on latest head to resolve conflicts.

File: config/arm.xacro [hunk 1]
```diff
@@ -7,67 +7,76 @@
...
     </xacro:macro>
 
     <!--VIRTUAL JOINT: Purpose: this element defines a virtual joint between a robot link and an external frame of reference (considered fixed with respect to the robot)-->
-    <virtual_joint name="virtual_joint" type="fixed" parent_frame="world" child_link="panda_link0" />
+    <virtual_joint name="virtual_joint" type="fixed" parent_frame="world" child_link="$(arg arm_id)_link0" />
 
     <xacro:macro name="collision" params="link">
         <!-- Enable (environmental) collisions of ${link}_sc -->
-        <disable_default_collisions link="${link}_sc" />
+        <!-- <disable_default_collisions link="${link}_sc" /> -->
```
Comment (rhaschke) [inline config/arm.xacro:38]: Why did you comment this out?

File: config/arm.xacro [hunk 2]
```diff
@@ -7,67 +7,76 @@
...
+        <disable_collisions link1="${link}" link2="$(arg arm_id)_link8" reason="Default" />
+        <!-- <disable_collisions link1="${link}_sc" link2="$(arg arm_id)_link0" reason="Default" />
+        <disable_collisions link1="${link}_sc" link2="$(arg arm_id)_link1" reason="Default" />
+        <disable_collisions link1="${link}_sc" link2="$(arg arm_id)_link2" reason="Default" />
+        <disable_collisions link1="${link}_sc" link2="$(arg arm_id)_link3" reason="Default" />
+        <disable_collisions link1="${link}_sc" link2="$(arg arm_id)_link4" reason="Default" />
+        <disable_collisions link1="${link}_sc" link2="$(arg arm_id)_link5" reason="Default" />
+        <disable_collisions link1="${link}_sc" link2="$(arg arm_id)_link6" reason="Default" />
+        <disable_collisions link1="${link}_sc" link2="$(arg arm_id)_link7" reason="Default" />
+        <disable_collisions link1="${link}_sc" link2="$(arg arm_id)_link8" reason="Default" /> -->
```
Comment (rhaschke) [inline config/arm.xacro:57]: These were removed upstream.

File: config/hand.xacro [hunk 3]
```diff
@@ -3,16 +3,17 @@
...
+        <disable_collisions link1="${finger}" link2="$(arg arm_id)_link1" reason="Default" />
+        <disable_collisions link1="${finger}" link2="$(arg arm_id)_link2" reason="Default" />
+        <disable_collisions link1="${finger}" link2="$(arg arm_id)_link3" reason="Default" />
+        <disable_collisions link1="${finger}" link2="$(arg arm_id)_link4" reason="Default" />
+        <disable_collisions link1="${finger}" link2="$(arg arm_id)_link5" reason="Default" />
+        <disable_collisions link1="${finger}" link2="$(arg arm_id)_link6" reason="Default" />
+        <disable_collisions link1="${finger}" link2="$(arg arm_id)_link7" reason="Default" />
+        <disable_collisions link1="${finger}" link2="$(arg arm_id)_link8" reason="Default" />
+        <disable_collisions link1="${finger}" link2="$(arg arm_id)_hand" reason="Default" />
+        <disable_collisions link1="${finger}" link2="$(arg arm_id)_hand_sc" reason="Default" />
```
Comment (rhaschke) [inline config/hand.xacro:16]: Here is also a superfluous line.

**Review by rhaschke — CHANGES_REQUESTED** [review]
Looks like you screwed everything when merging with HEAD. Please clean up.
My suggestion is to start from scratch, to be honest.

**Comment by Xiaoyu27** [comment]
Hi, I redid the pull request here #106 , you can close this pull request.

**Comment by v4hn** [comment]
> Hi, I redid the pull request here https://github.com/ros-planning/panda_moveit_config/pull/106 , you can close this pull request.

You could have simply force-pushed your git branch for this pull request instead...

**Pull request closed without merging**